internal/chilogger/chilogger.go

```diff
@@ -48,6 +62,10 @@ func Logger(l interface{}, name string) func(next http.Handler) http.Handler {
 	return nil
 }
 
+func isDebugLog(method string, path string) bool {
+	return method == http.MethodGet && path == "/health"
```

Comment:
In future we need to also ignore PUT requests that go from agents, it's flooding logs as well.

Reply:
following to this discussion, the PUT requests are important to be in "info" and not move to "debug"
We will change the logger to use the production configuration and will check how this reduce the "repeated logs"